tests/Api/DataFixtures/ORM/product_variant.yaml

```
@@ -0,0 +1,22 @@
+Sylius\Component\Core\Model\Product:
+    product:
+        fallbackLocale: en_US
+        currentLocale: en
+        code: MUG_SW
+
+Sylius\Component\Product\Model\ProductVariantTranslation:
+    productVariantTranslation:
+        name: "Mug"
+        locale: en_US
+        translatable: "@productVariant"
+
+Sylius\Component\Core\Model\ProductVariant:
+    productVariant:
```

The current convention in other files is to use snake case notation
```suggestion
    product_variant:
```